src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -163,20 +154,28 @@
                 </Binding>
             </Border.Visibility>
         </Border>
-
+        
         <Rectangle Name="nodeBackground"
                    Grid.Row="1"
                    Grid.ColumnSpan="3"
                    Grid.RowSpan="3"
-                   Canvas.ZIndex="10">             
+                   Canvas.ZIndex="10"> 
```

Comment:
This helps to control the opacity on Ports. now the ports are readable even when the frozen node is moved around the geometry.